test_scripts/Handling_VR_help_requests/commonVRhelp.lua

```diff
@@ -0,0 +1,329 @@
+end
+
+function m.addCommandParams(pN)
+  local out = {
+    cmdID = pN,
+    vrCommands = { "vrCommand" .. pN},
+    menuParams = {
+    menuName = "Command" .. pN
+    }
+  }
+  return out
+end
+
+function m.addCommand(pParams, pAppId)
+  if not pAppId then pAppId = 1 end
+  local mobSession = m.getMobileSession(pAppId)
+  local hmiConnection = m.getHMIConnection()
+  local cid = mobSession:SendRPC("AddCommand", pParams)
+  EXPECT_HMICALL("UI.AddCommand")
+  :Do(function(_,data)
+    hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  if pParams.vrCommands then
+    table.insert(m.commandArray, { cmdID = pParams.cmdID, vrCommand = pParams.vrCommands})
+    local requestUiParams = {
+      cmdID = pParams.cmdID,
+      vrCommands = pParams.vrCommands,
+      type = "Command",
+      appID = m.getHMIAppId(pAppId)
+    }
+    EXPECT_HMICALL("VR.AddCommand", requestUiParams)
+    :Do(function(_,data)
+      hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  end
+  mobSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+function m.deleteCommand(pParams, pVRInterface, pAppId)
+  if not pAppId then pAppId = 1 end
+  local mobSession = m.getMobileSession(pAppId)
+  local hmiConnection = m.getHMIConnection()
+  local cid = mobSession:SendRPC("DeleteCommand", pParams)
+  EXPECT_HMICALL("UI.DeleteCommand")
+  :Do(function(_,data)
+    hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  if pVRInterface then
+    for k, v in pairs(m.commandArray) do
+      if v.cmdID == pParams.cmdID then
+        table.remove(m.commandArray, k)
+      end
+    end
+    local requestVrParams = {
+      cmdID = pParams.cmdID,
+      type = "Command",
+      appID = m.getHMIAppId(pAppId)
+    }
+    EXPECT_HMICALL("VR.DeleteCommand", requestVrParams)
+    :Do(function(_,data)
+      hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  end
+  mobSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+function m.setGlobalProperties(pParams, pAppId)
+  if not pAppId then pAppId = 1 end
+  local mobSession = m.getMobileSession(pAppId)
+  local hmiConnection = m.getHMIConnection()
+  local cid = mobSession:SendRPC("SetGlobalProperties", pParams.requestParams)
+  if pParams.requestParams.vrHelp or
+    pParams.requestParams.keyboardProperties then
+    EXPECT_HMICALL("UI.SetGlobalProperties", pParams.requestUiParams)
+    :Do(function(_,data)
+      hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  end
+  if pParams.requestParams.helpPrompt or pParams.requestParams.timeoutPrompt then
+    EXPECT_HMICALL("TTS.SetGlobalProperties", pParams.requestTtsParams)
+    :Do(function(_,data)
+      hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  end
+  mobSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+function m.setGlobalPropertiesFromSDL(pTST)
```

Comment:
@GetmanetsIrina Since ```pTST``` parameter is boolean it's better to rename it to something comprehensible, e.g. ```pIsCheckOnTimeoutRequied```